protos/flyteidl/admin/launch_plan.proto

```diff
@@ -90,7 +91,10 @@ message LaunchPlanSpec {
     // Indicates the permission associated with workflow executions triggered with this launch plan.
     Auth auth                             = 8 [deprecated=true];
 
-    AuthRole auth_role                    = 9;
+    AuthRole auth_role                    = 9 [deprecated=true];
```

Comment:
can we undeprecate this now then?

Reply:
no. why? This is deprecated

Reply:
oh I see so `map<string, string> values ` in SecurityContext will contain these. nvm

Reply:
wow the deprecations